shatterhand2.yml

```diff
@@ -0,0 +1,70 @@
+---  # document start
```

Comment:
E tek sad YAML liči na nešto 😄 

Preimenuj fajl u nešto normalno... *shatterhand2.yml* mi ništa ne govori

Reply:
✔️